mex-wholebodymodel/library/src/mexmain.cpp

```diff
@@ -39,68 +35,56 @@ using namespace mexWBIComponent;
  * To properly destroy, call ComponentManager::deleteInstance()
  * and then set this pointer to 0.
  */
-static ComponentManager *componentManagerLocalPointerCopy = 0;
```

Comment:
I understand in MFC I used mostly the _nullptr_ instead of NULL or 0. But if _nullptr_ makes troubles  in Ubuntu 12 and 14, in the meantime we can go back to the old C++98 standard.
And _0_ can be changed always to _nullptr_ afterwards. But it is interesting to hear that there are some troubles in Ubuntu ... I will ask Mr. Google ... ;)